api/v1/hostprofile_types.go

```diff
@@ -854,6 +854,10 @@ type ProfileBaseAttributes struct {
 	// AppArmor defines the security model on the host.
 	// +optional
 	AppArmor *string `json:"appArmor,omitempty"`
+
+	// HwSettle defines the wait time for SCSI devices to show up.
+	// +optional
```

Comment:
If this parameter only takes number, could you consider to add validation annotation? (You could refer "MaxCPUMhzConfigured" above)

Reply:
yes, this parameter only takes number. added validation annotation.